recipes-core/images/xenclient-initramfs-image.bb

```diff
@@ -19,7 +19,11 @@ IMAGE_INSTALL = " \
     libtctidevice \
     tpm-tools-sa \
     tpm2-tools \
-    initramfs-xenclient \
```

Comment:
I think you'll want an additional commit to clean up the initramfs-xenclient recipe so it is only applicable to the stubdom.

Reply:
I would leave it for now since there is work that will result in us being able to just drop the whole recipe all together.